node-rcon.js

```diff
@@ -19,10 +19,15 @@ var PacketType = {
 };
 
 /**
- * options:
- *   tcp - true for TCP, false for UDP (optional, default true)
- *   challenge - if using UDP, whether to use the challenge protocol (optional, default true)
- *   id - RCON id to use (optional)
+ * Rcon, opens a RCON connection used by Valve products and GoldSrc, also implemented into the Minecraft Protocol
+ * @param {string} host - A String representing the IP address of the server you are trying to connect to
+ * @param {number} port - The RCON port of the server, configurable in the server.properties on the server
```

Comment:
For any game-specific documentation, it should be clear which game is being referred to (like you did for `cmd` in `send`).

Reply:
Alright, going to fix it soon!